Update to new standard

mueschm opened the pull request:
## Overview
Update to the new standard. This project mostly already conformed, so just updated some of the styling and the readme.

## Changes
1. Remove the big block of text on the site
2. Show movie form by default
3. Updated readme

## Pictures
### Local

<img width="1228" alt="chrome_EyyWtiUDpN" src="https://user-images.githubusercontent.com/532523/225761974-ff481a9e-6e92-4e64-99c8-46a3dfbb3224.png">

### Dev

<img width="1230" alt="chrome_6kGIaaWlnu" src="https://user-images.githubusercontent.com/532523/225762050-a6beff2f-bf5c-4be7-a9bf-3c0585042369.png">

Commit:
Update to new standard

ryan-cahill commented on README.md:
```diff
@@ -16,54 +16,113 @@
 
-This starter application will show how easy it is to deploy an application both
-locally and in a remote environment.
+# Go Starter Project
```
I hadn't realized that C# code looked so much like Go code

ryan-cahill commented on README.md:
```diff
@@ -16,54 +16,113 @@
+environment automatically. This allows you to quickly iterate and see your changes without having to restart the
+application stack.
+
+Give it a try! Search inside your project for “Favorite Movie” and change this string to "Favorite Pizza" Once you save
```
The "Favorite movie" header mentioned here isn't included in the template

ryan-cahill reviewed (changes requested):
Why is there an item seeded in the DB when the app starts? I think that could be confusing to some users

ryan-cahill commented:
Also in testing the app (noticed with running `architect dev --debug=false` if you submit the form, then refresh the page after, it keeps submitting the form over and over on page refresh even though there's also nothing in the text fields

Commit:
Fix issues

mueschm commented:
Updated
![image](https://user-images.githubusercontent.com/532523/226754479-d87862c3-0b8b-4b91-b44a-636fd832d843.png)

ryan-cahill commented:
Thanks for fixing those things. Two things left:

1. We added a DB liveness probe to the other templates which should be added here too
```
    liveness_probe:
      command: pg_isready -d ${{ secrets.database_name }} -U ${{ secrets.postgres_username }}
      interval: 10s
      failure_threshold: 3
```
2. I still think it's weird that there's an item seeded in the database when the app starts. It's even weirder that the rating is 10 when it's only supposed to be 1-5 according to the text field's placeholder

ryan-cahill reviewed (changes requested):
Thanks for fixing those things. Three things left:

1. We added a DB liveness probe to the other templates which should be added here too
```
    liveness_probe:
      command: pg_isready -d ${{ secrets.database_name }} -U ${{ secrets.postgres_username }}
      interval: 10s
      failure_threshold: 3
```
2. I still think it's weird that there's an item seeded in the database when the app starts. It's even weirder that the rating is 10 when it's only supposed to be 1-5 according to the text field's placeholder
3. The app doesn't deploy to a remote environment and fails with the following error
```
warn: Microsoft.AspNetCore.DataProtection.Repositories.FileSystemXmlRepository[60]
Storing keys in a directory '/root/.aspnet/DataProtection-Keys' that may not be persisted outside of the container. Protected data will be unavailable when container is destroyed.
warn: Microsoft.AspNetCore.DataProtection.KeyManagement.XmlKeyManager[35]
No XML encryptor configured. Key {a3577738-831d-47d3-8456-2d79191adfcc} may be persisted to storage in unencrypted form.
Finished configuring web server
info: Microsoft.Hosting.Lifetime[14]
Now listening on: http://[::]:5001
info: Microsoft.Hosting.Lifetime[0]
Application started. Press Ctrl+C to shut down.
info: Microsoft.Hosting.Lifetime[0]
Hosting environment: Production
info: Microsoft.Hosting.Lifetime[0]
Content root path: /app
info: System.Net.Http.HttpClient.api.LogicalHandler[100]
Start processing HTTP request GET https://api.example-environment.architect-ryan.dev.arcitect.io/movie
info: System.Net.Http.HttpClient.api.ClientHandler[100]
Sending HTTP request GET https://api.example-environment.architect-ryan.dev.arcitect.io/movie
fail: Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware[1]
An unhandled exception has occurred while executing the request.
System.Net.Http.HttpRequestException: The SSL connection could not be established, see inner exception.
---> System.Security.Authentication.AuthenticationException: The remote certificate is invalid because of errors in the certificate chain: NotTimeValid, UntrustedRoot
at System.Net.Security.SslStream.SendAuthResetSignal(ProtocolToken message, ExceptionDispatchInfo exception)
at System.Net.Security.SslStream.CompleteHandshake(SslAuthenticationOptions sslAuthenticationOptions)
at System.Net.Security.SslStream.ForceAuthenticationAsync[TIOAdapter](TIOAdapter adapter, Boolean receiveFirst, Byte[] reAuthenticationData, Boolean isApm)
at System.Net.Http.ConnectHelper.EstablishSslConnectionAsync(SslClientAuthenticationOptions sslOptions, HttpRequestMessage request, Boolean async, Stream stream, CancellationToken cancellationToken)
--- End of inner exception stack trace ---
at System.Net.Http.ConnectHelper.EstablishSslConnectionAsync(SslClientAuthenticationOptions sslOptions, HttpRequestMessage request, Boolean async, Stream stream, CancellationToken cancellationToken)
at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
at System.Net.Http.HttpConnectionPool.AddHttp11ConnectionAsync(HttpRequestMessage request)
at System.Threading.Tasks.TaskCompletionSourceWithCancellation`1.WaitWithCancellationAsync(CancellationToken cancellationToken)
at System.Net.Http.HttpConnectionPool.GetHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
at System.Net.Http.HttpConnectionPool.SendWithVersionDetectionAndRetryAsync(HttpRequestMessage request, Boolean async, Boolean doRequestAuth, CancellationToken cancellationToken)
at System.Net.Http.DiagnosticsHandler.SendAsyncCore(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)
at Microsoft.Extensions.Http.Logging.LoggingHttpMessageHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
at Microsoft.Extensions.Http.Logging.LoggingScopeHttpMessageHandler.SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
at System.Net.Http.HttpClient.g__Core|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)
at asp_net_core_web.Pages.IndexModel.OnGet() in /app/asp-net-core-web/Pages/Index.cshtml.cs:line 42
at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.ExecutorFactory.NonGenericTaskHandlerMethod.Execute(Object receiver, Object[] arguments)
at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeHandlerMethodAsync()
at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeNextPageFilterAsync()
at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.Rethrow(PageHandlerExecutedContext context)
at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
at Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure.PageActionInvoker.InvokeInnerFilterAsync()
at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.g__Awaited|25_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
at Microsoft.AspNetCore.Routing.EndpointMiddleware.g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.g__Awaited|6_0(ExceptionHandlerMiddleware middleware, HttpContext context, Task task)
```

Commit:
Fixed

mueschm commented:
1. Liveness probe is in. Added to others but forget here so good catch.
2. Removed the item so it does not show up.
3. You are seeing that because ASP.net by default requires a valid SSL cert to run. You can disable that but as our default behavior is to add SSL certs to everything it made no sense. I assume you deployed to staging or local since my production deploy works fine.

ryan-cahill reviewed (approved):
`architect dev`
![image](https://user-images.githubusercontent.com/8558567/227015089-bffc8989-253e-45f6-9a5f-961c87679ac5.png)

`architect dev --debug=false`
![image](https://user-images.githubusercontent.com/8558567/227015568-a6b5f144-b2a9-4916-ab06-de7fb2420e41.png)

`architect deploy asp-net-core -e example-environment --auto-approve`
![image](https://user-images.githubusercontent.com/8558567/227016658-6550c35a-f270-4125-8364-91fb793830e6.png)

zach-j-dev reviewed (approved):
Looks good from my end! Everything runs smoothly and works as expected
